### DISCUSSION TITLE
Training mode is not supported in this build.

### DISCUSSION THREAD

**Discussion by divisonofficer** [RapidAI/RapidLatexOCR#22] [Q&A] [answered]
请提供下述完整信息以便快速定位问题
(Please provide the following information to quickly locate the problem)
- **系统环境/System Environment**: Ubuntu 20.4 (WSL 2)
- **使用的是哪门语言的程序/Which programing language**: Python3.8
- **所使用语言相关版本信息/Version**: 0.6
- **OnnxRuntime版本/OnnxRuntime Version**: 1.8
- **使用`rapidocr_onnxruntime`版本**/Use `rapidocr_onnxruntime` version: 1.3.16
- **可复现问题的demo和文件/Demo of reproducible problems**: import LatexOcr, create instance of LatexOCR, run model

![image](https://github.com/RapidAI/RapidLaTeXOCR/assets/41609506/7f2a6044-dc8f-4128-86fb-62e86ca1f6d5)

- **完整报错/Complete Error Message**:
onnxruntime.capi.onnxruntime_pybind11_state.RuntimeException: [ONNXRuntimeError] : 6 : RUNTIME_EXCEPTION : Exception during initialization: /onnxruntime_src/onnxruntime/core/providers/cpu/nn/batch_norm.h:52 onnxruntime::BatchNorm<T>::BatchNorm(const onnxruntime::OpKernelInfo&) [with T = float] !is_train_ was false. Training mode is not supported in this build.
 
- **可能的解决方案/Possible solutions**:

How to disable Training mode?

**Comment by SWHL** [comment, marked as answer]
I tested the program in Google Colab, and do not meet the problem.
Can you try to test it on the [Google Colab](https://colab.research.google.com/notebooks/welcome.ipynb#scrollTo=vR2GwbsQNnSS)? 
I want to ensure that the error can be reproduce.

For example code:
```bash
pip install rapid_latex_ocr
```

```python
from rapid_latex_ocr import LatexOCR

model = LatexOCR()

img_path = "6.png"
with open(img_path, "rb") as f:
    data = f.read()

res, elapse = model(data)

print(res)
print(elapse)
```
